File: applications/summersim18/src/cpp/mul.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include <float.h>
#include <iostream>
#include <iomanip>

int main()
{
    double a,b,c,d;
    a = 0.001;
    b = 1e308;
    c = 11;

    d = (a * b) * c;
    std::cout << "(0.001 * 1e308) * 11 = " << std::setprecision(20) << d << std::endl;

    d = a * (b * c);
    std::cout << "0.001 * (1e308 * 11) = " << std::setprecision(20) << d << std::endl;

    d = a * b * c;
    std::cout << "0.001 * 1e308 * 11 = " << std::setprecision(20) << d << std::endl;

    return 0;
}

File: applications/summersim18/src/python/mul.py
if __name__ == '__main__':
    a = 0.001
    b = 1e308
    c = 11
    d = (a * b) * c
    print("(0.001 * 1e308) * 11 = %01.20e" % d)

    d = a * (b * c)
    print("0.001 * (1e308 * 11) = %01.20e" % d)

    d = a * b * c
    print("0.001 * 1e308 * 11 = %01.20e" % d)
